omod/src/main/webapp/resources/jsonforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.json

```diff
@@ -1135,6 +1138,21 @@
                   }
                 ]
               }
+            },
```

Comment:
@slubwama 
Did you pull the latest changes from master.

There's already this code block on the assessment form:
`        {
          "label": "Return Visit Details",
          "isExpanded": "true",
          "questions": [
            {
              "label": "Return date",
              "type": "obs",
              "required": "true",
              "id": "returnDate",
              "questionOptions": {
                "rendering": "date",
                "concept": "dcac04cf-30ab-102d-86b0-7a5022ba4115"
              },
              "validators": []
            },
            {
              "label": "Reasons for next appointment",
              "type": "obs",
              "required": false,
              "id": "reasonForNextAppoinment",
              "questionOptions": {
                "rendering": "multiCheckbox",
                "concept": "160288AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
                "answers": [
                  {
                    "concept": "ab4510ac-3feb-4abb-8653-5d252813798f",
                    "label": "ART Initiation"
                  },
                  {
                    "concept": "e06ac289-2920-4b85-80c9-805c69b6e5a9",
                    "label": "ART Refill"
                  },
                  {
                    "concept": "4b19b253-97c4-4b1b-bb14-41f2c576b8c5",
                    "label": "ART Regimen Switch/ Substitution"
                  },
                  {
                    "concept": "dc8d83e3-30ab-102d-86b0-7a5022ba4115",
                    "label": "HIV VIRAL LOAD"
                  },
                  {
                    "concept": "dcbcba2c-30ab-102d-86b0-7a5022ba4115",
                    "label": "CD4 COUNT"
                  },
                  {
                    "concept": "163153AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
                    "label": "Intensive adherence counseling sessions set"
                  },
                  {
                    "concept": "5def0b16-460a-494b-ab1f-ac8e398a88fa",
                    "label": "T.B Drug Refill"
                  },
                  {
                    "concept": "801b8959-4b2a-46c0-a28f-f7d3fc8b98bb",
                    "label": "ANC Visit"
                  },
                  {
                    "concept": "3f0f9c04-25a4-4070-84cc-5b49566aeb4c",
                    "label": "EID Visit"
                  },
                  {
                    "concept": "dcd7e8e5-30ab-102d-86b0-7a5022ba4115",
                    "label": "PMTCT"
                  },
                  {
                    "concept": "dc8f22f9-30ab-102d-86b0-7a5022ba4115",
                    "label": "HYPERTENSION"
                  },
                  {
                    "concept": "dcbc7b8e-30ab-102d-86b0-7a5022ba4115",
                    "label": "NUTRITIONAL SUPPORT"
                  },
                  {
                    "concept": "32dbfbc2-0f6c-4beb-88d0-55e0ee1a7b8c",
                    "label": "CLINICAL ASSESSMENT"
                  },
                  {
                    "concept": "347e93a0-72c2-400f-9579-116ce3ba6a68",
                    "label": "Checkup"
                  }
                ]
              },
              "validators": []
            }
          ]
        }`

Reply:
@Daphne210 I think this has to do with date when a patient will return to the facility not next appointment date. These two concepts are quite different. One is to determine patients who go out  and receive services out of the facility